cmd/build.go

```diff
@@ -27,7 +27,7 @@ type buildCmd struct {
 
 type buildOpts struct {
 	config        string
-	id            string
```

Comment:
should probably rename this to `ids`

Reply:
then it would be backwards incompatible and that's bad. More over since you add the ids one by one (--id id1 --id id2) it makes sense for it to be singular

Reply:
the name of the arg can still be `id`, we can safely change name of the field in the struct